reference/forms/types/options/form_attr.rst.inc

```diff
@@ -0,0 +1,26 @@
+``form_attr``
+~~~~~~~~~~~~~
+
+**type**: ``boolean`` or ``string`` **default**: ``false``
+
+When ``true`` and used on a form element, it adds a `"form" attribute`_. to its HTML field representation with
+its HTML form id. By doing this, form element can be rendered outside HTML form while still working as expected::
+
+    $builder->add('body', TextareaType::class, [
+        'form_attr' => true,
+    ]);
+
+This can be useful when you need to solve nested form problems.
+You can also set this to ``true`` on a root form to automatically set the "form" attribute on all its children.
+
+.. note::
+
+    When root form has no ID, ``form_attr`` required to be a string identifier to be used as form ID.
+
+.. caution::
+
+    HTML "form" attribute was introduced to let you explicitly bind a control with a form even if it is
+    not nested inside it. Don't forget to check for `browser compatibility`_.
```

Comment:
Do we really need the `caution` block?

Reply:
Browsers not supporting this are really old and we could remove the caution block. I thought to keep this just because of IE, which is not supported. What do you think?

Reply:
Let's remove it IMO. Developers who still need to support IE in a way are likely aware of the fact that they need to be extra careful to check which features are compatible.

Reply:
Ok, I agree